generators/app/index.js

```diff
@@ -138,6 +186,7 @@ module.exports = class extends Generator {
       this.props,
     );
 
+    // Form files
```

Comment:
should this actually be `non-form files`?

Reply:
Er...only if you wanna make sense... :homer-disappear: